input/fsh/SD_Genomics.fsh

```diff
@@ -235,20 +235,22 @@ Description:    "The area of the genome region referenced in testing for variant
 * component[genomicReferenceSequenceId] ^definition = "Range(s) of DNA sequence examined. The genomic reference sequence is a contiguous stretch of chromosome DNA that spans all of the exons of the gene and includes transcribed and non transcribed stretches. For this ID use either the NCBI genomic nucleotide RefSeq IDs with their version number (see: NCBI.NLM.NIH.Gov/RefSeq) or use the LRG identifiers, without transcript (t or p) extensions -- when they become available. (source: LOINC)"
 * component[genomicReferenceSequenceId].code = LNC#48013-7
 
-// * insert ReduceText
-// * insert ReduceText(referenceRange)
-// * insert ReduceText(component)
-// These are the additional MS on top of US Core Lab Observation
 * component and component.code and component.value[x] and component.dataAbsentReason MS
 
+Profile:    GenomicDiagnosticImplication
+Parent:     USCoreObservationLab
+Id: mcode-genomic-diagnostic-implication
+Title:      "Genomic Diagnostic Implication"
+Description:    "Observation stating a linkage between one or more genotype/haplotype/variation Observations and evidence for or against a particular disease, condition, or cancer diagnosis."
+* code = LNC#81259-4 "Associated phenotype" //"Associated phenotype"
+* derivedFrom only Reference(CancerGeneticVariant)
+* value[x] 0..0 
+* value[x] ^definition = "Not used in this profile. MustSupport is inherited from the parent profile (USCoreObservationLab) and should be ignored by implementers for this element."
```

Comment:
This is problematic. I don't think you can wave away a MS. Perhaps another reason to reconsider the resource type or parent?

Reply:
Again, I was following the CG Reporting IG pattern in which diagnostic-implication is based on a parent called genomic-implication which zero's out value[x] (http://build.fhir.org/ig/HL7/genomics-reporting/StructureDefinition-implication.html), however that doesn't necessarily make their profile correct or at least optimal.

For now, I'll remove the 0..0 constraint and bring up the issue of 0..0 value at the next CGWG modeling subcommittee meeting. 
I can enter a JIRA ticket against their balloted IG and request them to open this up - maybe make it simply fixed to "present", meaning that if any observation that is an implication that exists and is derived from a Variant is asserted as present.

On a related note, I incorrectly defined code in this profile. It's actually a TBD Code in CGWG and fixed to diagnostic-implication with the CGWG needing to submit a request to LOINC to add. I'll push this in the commit. I'll follow-up with either Joel or Patrick from CGWG on the status of getting a LOINC code for that.
I also changed the derivation from USCoreLabObservation to the base Observation since diagnostic-implication is more of an evaluation of the associated variant and not an actual lab result in itself.
Changes pushed in commit https://github.com/HL7/fhir-mCODE-ig/pull/198/commits/3b786dbd63219b69863df75a43d815f4199af69c

Reply:
On more comment on line 246 -- assuming Observation is what we go with, I think the variant should be the focus of the observation, not derivedFrom